src/main/java/in/projecteka/consentmanager/consent/model/HIPReference.java

```diff
@@ -17,6 +16,6 @@
 public class HIPReference implements Serializable {
     @NotEmpty(message = "HIP identifier is not specified.")
     private String id;
-    @JsonInclude(Include.NON_NULL)
+    @Nullable
     private String name;
```

Comment:
you can just remove this field. this is not needed

Reply:
As I told you, It's been using in some other places